### PR TITLE
Adding message field for SqueezeDet loss

### PR DESCRIPTION
**Pull request by kvmanohar22**
In Ref: #764 
I've added the hyper-parameters which should be mentioned while using this loss function.

### CODE REVIEW

**Comment by bhack** [comment]
@edgarriba @karandesai-96 Need we also to sync generally the caffe proto again from upstream?

**Comment by edgarriba** [comment]
@kvmanohar22 have you tried this? is there any way that you can provide a test for this?

**Comment by kvmanohar22** [comment]
@edgarriba I haven't tested this. But after reading the paper mentioned in #764, I felt these fields would be required in order to compute the loss.

**Comment by edgarriba** [comment]
@kvmanohar22 we cannot merge without tests

**Comment by edgarriba** [comment]
@kvmanohar22 any progress on that? close this and reopen once you have something more complet

**Comment by kvmanohar22** [comment]
@edgarriba I'm infact working on the loss function for caffe, so I guess we can close this.
